.github/workflows/tests.yaml

```diff
@@ -8,8 +8,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Build container
         run: docker build -t pdudaemon-ci -f Dockerfile.dockerhub .
-      - name: Run pycodestyle
-        run: docker run --rm -v $(pwd):/p -w /p pdudaemon-ci /root/.local/bin/pycodestyle --config=pycodestyle_config .
+      - name: Run lint
+        run: docker run --rm -v $(pwd):/p -w /p pdudaemon-ci /root/.local/bin/pre-commit run --all-files
```

Comment:
if the pre-commit.ci github integration is used, so it runs on each commit/PR, you can omit the lint step for the github actions if you prefer

Reply:
As https://github.com/pdudaemon/pdudaemon/pull/134 uses the same Dockerfile to build containers intend for users, we shouldn't add git and pre-commit specific configuration to it. So it seems a  separate pre-commit GitHub workflow would be better (or what do you mean by intregration?).

Reply:
The integration I am referring to is utilizing the (free for open source) service: https://[pre-commit.ci](https://pre-commit.ci/)/. It works similarly to a GitHub Action

Reply:
No issues, I tried to follow the existing convention where `pycodestyle` was inside the docker container, do instruct me otherwise if you have a preference on an alternative.